examples/refs.rs

```diff
@@ -47,14 +46,13 @@ fn show_first(c: impl DisplayCollection) {
     }
 }
 
-
 fn main() {
     let v = vec!["dog", "cat"];
     let boxed = Box::new(v.clone());
 
-    show_first(v.clone());      // Vec<&str>    (our manual impl)
-    show_first(&v);             // &Vec<&str>   (auto-impl)
-    show_first(&&v);            // &&Vec<&str>  (works too, of course)
-    show_first(boxed.clone());  // Box<Vec<&str>> (auto-impl)
-    show_first(&boxed);         // &Box<Vec<&str>>
+    show_first(v.clone()); // Vec<&str>    (our manual impl)
+    show_first(&v); // &Vec<&str>   (auto-impl)
+    show_first(&&v); // &&Vec<&str>  (works too, of course)
+    show_first(boxed.clone()); // Box<Vec<&str>> (auto-impl)
+    show_first(&boxed); // &Box<Vec<&str>>
```

Comment:
This alignment was also on purpose, so please revert it to how it was.

Reply:
I guess I need to setup my rustfmt properly :) Will do